core/Makefile

```diff
@@ -1,34 +1,42 @@
-dev-discovery-1::
-	@go run main.go -env-file=./infra/dev_config/discovery-one.env
-
-dev-content-1::
-	@go run main.go -env-file=./infra/dev_config/content-one.env
-
-dev-content-2::
-	@go run main.go -env-file=./infra/dev_config/content-two.env
+SQL_SRCS := $(shell find db/sql/ -type f -name '*.sql') db/sqlc.yaml
```

Comment:
what is this for if line 24 is unchanged?

Reply:
This is for making the gen target only run if the sql files have been modified (line [18](https://github.com/AudiusProject/audius-protocol/pull/9300/files?diff=split&w=0#diff-d486e601a844f08eaa052d963bedbdd8a7db76d664a5f44fe34bf4fd6db1064aR18))